examples/hooks/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { useBreadcrumb, UseBreadcrumbProps } from 'react-instantsearch-hooks';
+
+import { cx } from '../cx';
+
+export type BreadcrumbProps = React.ComponentProps<'div'> & UseBreadcrumbProps;
+
+export function Breadcrumb(props: BreadcrumbProps) {
+  const { items, refine, createURL } = useBreadcrumb(props);
+
+  return (
+    <div
+      className={cx(
+        'ais-Breadcrumb',
+        items.length === 0 && 'ais-Breadcrumb--noRefinement',
+        props.className
+      )}
+    >
+      <ul className="ais-Breadcrumb-list">
+        <li
+          className={cx(
+            'ais-Breadcrumb-item',
+            items.length === 0 && 'ais-Breadcrumb-item--selected'
+          )}
+        >
+          <a
+            href={createURL(null)}
+            onClick={(event) => {
+              event.preventDefault();
+              refine(null);
```

Comment:
We've been using `undefined` to reflect that the refined value must be unset.

```suggestion
              refine(undefined);
```